### ISSUE TITLE
Different Install button - Small Blue with Dropdown option

### ISSUE THREAD

**Issue by killuhwhale** [killuhwhale/amace_validator#9]
TL;DR
"blue install buttons appears randomly and the app can be installed or may be incompatible...." but seems to always fail when it appears.

A blue install button with a drop down menu on the right hand side shows sometimes.

This seems to replace the text there is an incompatibility; allowing the user to select the dropdown to install on another device.

Still not sure whats happening.

It is either:
1. Clicking the button but fails to install
2. It fails to find the install button

Need to verify the behavior and when this button appears.
But I think it appears when the app is a valid install still....

It is just hard to reproduce and get that button to show.

How the issue was discovered:

1. Ran 500 Apps
2. 85 Apps failed to install
3. Reran 85 apps
4. Found that they failed for:
    - Too long to install
    - Compat issue
5. However, some of the 85 apps were good and installed during the retry,
6. This means, the blue install buttons appears randomly and the app can be installed or may be incompatible....

**Comment by killuhwhale** [comment]
This should be handled since there is a UI check for a View with Cont-Description "Install"

Verifying this is working as intended...

**Comment by killuhwhale** [comment]
Blue Button is being clicked to install.

Not sure about 2 things:
1. If the blue button appears when an app isnt actually able to be installed
2. If it does appear, not sure the results of clicking install button; most likely will fail due to TooManyAttempts


Closing for now. Please reopen if apps fails to install when the Blue Install w/ Drop Down appears.
